jscomp/build_tests/super_errors/expected/bigint_match_literal.res.expected

```diff
@@ -0,0 +1,11 @@
+
+  [1;33mWarning number 11[0m
```

Comment:
just todo note. maybe it will be better to disable ansi color for test snapshot, or to checkin both colored and not